sdk.go

```diff
@@ -114,12 +116,22 @@ func Insecure(insecure bool) ServeOption {
 	}
 }
 
+// MaxMsgSize returns a ServeOption to set the max message size in bytes the server can receive.
```

Comment:
Ah, looks like this needs a slight tweak to appease the linter. 😄

Reply:
I modified the comment... :D

Reply:
Thank you! Sadly it looks like this needs another rebase, and DCO is failing again. ☹️ Sorry this small change has been such a pain to get in!

Reply:
Should have fixed it, or at least i hope! Sorry for the annoyment...